src/flicking.js

```diff
@@ -1354,7 +1354,7 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 			}
 
 			this._conf.indexToMove = indexToMove;
-			this._setValueToMove(isPositive);
+			this._setValueToMove(indexToMove > 0);
```

Comment:
ok. this code is so confusing.

How about define `var isPositive` in `circular` case?
```js
if (circular) {
  indexToMove = no - panel.no;
  var isPositive = indexToMove > 0;		 				
// ...
}
```